openslides/config/views.py

```diff
@@ -100,3 +101,22 @@ def form_valid(self, form):
             config[key] = form.cleaned_data[key]
         messages.success(self.request, _('%s settings successfully saved.') % _(self.config_collection.title))
         return super(ConfigView, self).form_valid(form)
+
+
+class ConfigViewSet(viewsets.ViewSet):
```

Comment:
This could be a problem. What permission does a user need to see all config values?

Could there be config values which are secret? Maybe in any plugin or for a short period of time? For example who voted for who